### ISSUE TITLE
New Pluto Package manager makes everything run VERY slowly on clusters.

### ISSUE THREAD

**Issue by natgeo-wong** [fonsp/Pluto.jl#1341]
Due to the nature of HPC/clusters and their storage management, Julia package management is often very slow on clusters (because clusters don't do well handling a large number of small files).  Therefore, the new Pluto Package manager makes the creation of new notebooks extremely slow on clusters.  Out of necessity, I would like to request that we be allowed to disable PlutoPkg on startup of a Pluto session, rather than doing it simply in notebooks.

It takes me like 5 minutes or something (i.e. the creation of this entire issue) to create a new notebook, whereas in old versions I could do it in seconds.

**Comment by j-fu** [comment]
This would also help with  developing a Pluto notebook as part of package documentation in parallel with the development of the package itself. In the moment I do weird things like this: https://github.com/j-fu/VoronoiFVM.jl/blob/25dc8d20606091c4cc45fd470082786085834a70/pluto-examples/nbproto.jl#L7 to disable PlutoPkg when working in the dev'ed package directory. 

Probably something like  Pluto.run(project=... ) would be sufficient here. Also, for batch generating html docs from Pluto notebooks using a not registered yet version of the package, a similar option to PlutoSliderServer would be nice.

**Comment by fonsp** [comment]
Hi @j-fu ! Can you open a new discussion or issue about this? I think the other pluto contributors might have some ideas.

**Comment by fonsp** [comment]
The original issue has been solved by Julia 1.7! See https://julialang.org/blog/2021/11/julia-1.7-highlights/#improved_performance_for_handling_registries_on_windows_and_distributed_file_systems

After upgrading Julia, you might need to remove the old registry to be able to take advantage of the performance boost:

```julia
julia> general_path = joinpath(DEPOT_PATH[1], "registries", "General")
"/Users/fons/.julia/registries/General"

julia> rm(general_path; recursive=true)
```
